dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
@tdonohue , I took a closer look at the usage of the GroupService.create(Context) method.  This method is called. 
 Outside of unit test files, setName() seems to be called consistently after the create method.  

Would it make sense to set a default string like "New Group" within the create(Context) method to provide a default value in the case where the sort is executed?